test/inductor/test_compiled_autograd.py

```diff
@@ -172,6 +172,26 @@ def fn():
 
         self.check_output_and_recompiles(fn)
 
+    def test_graph_break_custom_op(self):
```

Comment:
curious: does it mean we need to graph-break when custom op is used in compiled autograd graph?

(Traceable FSDP2 requires that compiled autograd backward graph has no graph break, so hoping graph-break here is not a requirement haha)

Reply:
> curious: does it mean we need to graph-break when custom op is used in compiled autograd graph?

No. We are testing what happens if (1) a user has a custom op that graph breaks in the forward and (2) we use compiled autograd on this